config/initializers/cors.rb

```diff
@@ -1,5 +1,5 @@
 Barong::App.define do |config|
   config.set(:api_cors_origins, '*')
-  config.set(:api_cors_max_age, '3600')
-  config.set(:api_cors_allow_credentials, 'false')
+  config.set(:api_cors_max_age, '3600', regex: /^\d+$/)
+  config.set(:api_cors_allow_credentials, 'false', values: %w(false true))
```

Comment:
maybe we should convert as a boolean after?

Reply:
Yes, I thought about `type: Boolean`